presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -81,10 +81,14 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     -XX:+ExplicitGCInvokesConcurrent
     -XX:+HeapDumpOnOutOfMemoryError
     -XX:OnOutOfMemoryError=kill -9 %p
+    -Djmx.rmiserver.port=7199
```

Comment:
This is not the right way to pass this option. It should go in the `config.properties` file.